src/vs/base/browser/ui/list/listView.ts

```diff
@@ -552,9 +552,9 @@ export class ListView<T> implements ISpliceable<T>, IDisposable {
 
 		const uri = this.dnd.getDragURI(item.element);
 		item.dragStartDisposable.dispose();
+		item.row.domNode!.draggable = !!uri;
```

Comment:
The list was failing to unset an item as draggable, if all of a sudden the dnd provider would return `null` as a drag URI. We need this for converting a draggable item to a non-draggable one.